src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-columns;
+  max-width: (($day-size + $gap-columns) * $columns - $gap-columns);
```

Comment:
The calculation for 'max-width' is correct, but it's a complex expression that could be simplified for better readability. Consider calculating this value in a separate variable at the top of the file and then using that variable here.